starter-code/iteration-2/todoInterface.ts

```diff
@@ -3,3 +3,12 @@
 // - Method for adding task where will receive a string, doesn't return anything.
 // - Method for listing all task in the console, doesn't return anything.
 // - Method for delete a task where will receive a string, doesn't return anything.
+interface TodoInterface {
```

Comment:
you can export directly the whole interface while declaring it 👍 
`export interface TodoInterface {
`